### ISSUE TITLE
Universal Binary for macOS 11

### ISSUE THREAD

**Issue by thebitstick** [dmarcotte/easy-move-resize#60]
Works great under Rosetta 2, so no rush!

**Comment by dmarcotte** [comment]
Thanks for this @thebitstick!  I've not taken the macOS 11 plunge yet, so this helps.

**Comment by thebitstick** [comment]
I just tested compilation and it, uh, just works.
👍
Here's what Xcode 12.3 Beta gives me
[easy-move-resize 2020-12-06 18-43-40.zip](https://github.com/dmarcotte/easy-move-resize/files/5649874/easy-move-resize.2020-12-06.18-43-40.zip)
You could just do a `$ file` on the executable just to see the architectures, if you don't want to try running it.

**Comment by kamzata** [comment]
> I just tested compilation and it, uh, just works.
> 👍
> Here's what Xcode 12.3 Beta gives me
> [easy-move-resize 2020-12-06 18-43-40.zip](https://github.com/dmarcotte/easy-move-resize/files/5649874/easy-move-resize.2020-12-06.18-43-40.zip)
> You could just do a `$ file` on the executable just to see the architectures, if you don't want to try running it.
> 
> Device: Mac Mini (2020), with Apple Silicon M1, running macOS 11.1 Beta 2.

Just tested on Macbook Air M1 and it works. It's the first time I run this utility but I noticed that when I resize a window in the "normal" way it goes smoother than when I use the keys + the right mouse button. Did you notice the same?

**Comment by thebitstick** [comment]
Yeah I've always noticed that, even on my Ryzentosh system running Mojave.

**Comment by benoitvinc** [comment]
Same here. Happy to see i'm not the only one !!!

**Comment by thebitstick** [comment]
Can a version bump be done? All it takes is just a recompile on Xcode 12.

**Comment by dmarcotte** [comment]
Done! https://github.com/dmarcotte/easy-move-resize/releases/tag/1.4.1
